### PR TITLE
[JMRBA-37] - fixed install pipeline

### PR DESCRIPTION
**Pull request by Evange95**

### CODE REVIEW

File: src/controller/HelloWorldController.hpp [hunk 1]
```diff
@@ -15,46 +15,25 @@
...
-    // general
-    info->summary = "Say hello";
-    info->addResponse<HelloDto::ObjectWrapper>(Status::CODE_200, "application/json");
-    info->addResponse<String>(Status::CODE_404, "text/plain");
-    // params specific
-    info->pathParams["username"].description = "Username";
-  }
-  
-  ENDPOINT("GET", "/hello/{username}", hello) {
+  ENDPOINT("GET", "/", root) {
```
Comment (Danielecina) [inline src/controller/HelloWorldController.hpp]: register /hello

File: test/app/MyApiTestClient.hpp [hunk 2]
```diff
@@ -15,26 +16,30 @@
...
+/**
+ * Test API client.
+ * Use this client to call application APIs.
+ */
+class MyApiTestClient : public oatpp::web::client::ApiClient {
 
-  API_CLIENT_INIT(HelloApiTestClient)
+  API_CLIENT_INIT(MyApiTestClient)
 
   API_CALL("GET", "/", getRoot)
```
Comment (Danielecina) [inline test/app/MyApiTestClient.hpp]: use /hello

**Review by Danielecina — APPROVED** [review]

File: src/AppComponent.hpp [hunk 3]
```diff
@@ -15,34 +15,49 @@
...
 class AppComponent {
 public:
-
+  
+  /**
+   *  Create ConnectionProvider component which listens on the port
+   */
   OATPP_CREATE_COMPONENT(std::shared_ptr<oatpp::network::ServerConnectionProvider>, serverConnectionProvider)([] {
-    return oatpp::network::server::SimpleTCPConnectionProvider::createShared(PORT);
+    return oatpp::network::tcp::server::ConnectionProvider::createShared({"0.0.0.0", 8000, oatpp::network::Address::IP_4});
```
Comment (fredmaggiowski) [inline src/AppComponent.hpp]: Can you keep the port in a constant, it should be provided via Env Var (also let's use 3000 which si our standard port)

File: src/controller/StatusController.hpp [hunk 4]
```diff
@@ -1,71 +0,0 @@
-/*
```
Comment (fredmaggiowski) [inline src/controller/StatusController.hpp]: Let's keep the status controller that provides health and readiness routes for k8s

**Review by fredmaggiowski — CHANGES_REQUESTED** [review]
Also can you please add the `.mia-template` directory with interpolated README for the developer who creates a service from this example?

File: src/AppComponent.hpp (hunk 3 above)
Comment (Evange95) [reply src/AppComponent.hpp]: Providing it by Env Var seems too tricky for the actual usage. I put it in a costant

File: .mia-template/README.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+```bash
+  mkdir build && cd build
+  cmake ..
+  make
+  ./mia-platform-cpp-helloworld-exe  
+
+```
+
+By default the service will run on port 3000, to change the port please set `HTTP_PORT` env variable
```
Comment (fredmaggiowski) [inline .mia-template/README.md]: This is not taken from environemnt variables, meaning that this statement is false, isn't it?

File: src/App.cpp [hunk 6]
```diff
@@ -60,8 +55,8 @@ int main(int argc, const char * argv[]) {
 
   run();
   
-  // Print how much objects were created during app running, and what have left-probably leaked
-  // Disable object counting for release builds using '-D OATPP_DISABLE_ENV_OBJECT_COUNTERS' flag for better performance
+  /* Print how much objects were created during app running, and what have left-probably leaked */
```
Comment (fredmaggiowski) [inline src/App.cpp]: Let's keep the single lines comments or properly use a multiline 

```suggestion
  // Print how much objects were created during app running, and what have left-probably leaked
```

File: src/App.cpp [hunk 7]
```diff
@@ -60,8 +55,8 @@ int main(int argc, const char * argv[]) {
...
+  /* Disable object counting for release builds using '-D OATPP_DISABLE_ENV_OBJECT_COUNTERS' flag for better performance */
```
Comment (fredmaggiowski) [inline src/App.cpp]:
```suggestion
  // Disable object counting for release builds using '-D OATPP_DISABLE_ENV_OBJECT_COUNTERS' flag for better performance
```

**Review by fredmaggiowski — APPROVED** [review]

**Pull request merged**